content/pages/pricing.html

```diff
@@ -127,7 +128,7 @@ <h2 class="ui small monospace center aligned header">
                   {{ pricing.plan_feature("Concurrent builds", "2 concurrent builds", "fa-rectangle-vertical-history fa-swap-opacity") }}
                   {{ pricing.plan_feature("Public projects", "Host public documentation", "fa-globe") }}
                   {{ pricing.plan_feature("Public repositories", "Clone public repositories", "fa-globe") }}
-                  {{ pricing.plan_feature("Custom domain", "Use your own domain", "fa-at") }}
+                  {{ pricing.plan_feature("Custom domain", "2 custom domains", "fa-at") }}
```

Comment:
Do we have the number of projects using more than one domain on .org?

Reply:
We have 102 projects with more than 2 domains.

```
In [2]: Project.objects.annotate(n_domains=Count("domains")).filter(n_domains__gt=2).count()
Out[2]: 102
```

First 15 projects with many domains:

```
In [15]: list(Project.objects.annotate(n_domains=Count("domains")).filter(n_domains__gt=2).order_by("-n_domains")[:15].values_list("slug", flat=True))
Out[15]: 
['www',
 'readthedocs',
 'gooddal',
 'thiet-ke-web',
 'trevorbramwellnet',
 'astro-thesis',
 'stdlib-examples',
 'ompi',
 'zfsbootmenu',
 'python3info',
 'workshop-devops',
 'pirateweather',
 'astronaut-training-program',
 'jquantlib',
 'docoftheyear']
```